routes/employee_auth.js

```diff
@@ -95,6 +95,45 @@ const create_employee = async (employee) => {
     }
 }
 
+// Function which creates access token
+const create_access_token = (existing_employee) => {
+    const access_token = jwt.sign({ id: existing_employee.employee_id, admin: existing_employee.employee_is_admin }, config.access_token_secret, { expiresIn: '30s' });
```

Comment:
expire time is really low. is this just for testing?

Reply:
Yeah, just for testing

Reply:
For this commit let's set it to 15 minutes. As you are developing/testing this feature maybe then you can set it back to a lower value.

Or maybe we can include this in our config so that developing/testing values do not need to be committed.

What do you think would be the best approach?

Reply:
I've set the expire time to 900s(15min) and put it to config.js file. Please check the latest commit.